File: 코딩 기초 트레이닝/5명씩-p181886/5명씩.js
// https://school.programmers.co.kr/learn/courses/30/lessons/181886

function solution(names) {
    var answer = [];

    let temp = [];
    for(let i = 0 ; i < names.length ; i++) {
        temp.push(names[i]);

        if(temp.length === 5) {
            answer.push(temp[0]);
            temp = [];
        }
    }

    return temp.length === 0 ? answer : [...answer, temp[0]];
}

File: 코딩 기초 트레이닝/5명씩-p181886/5명씩.py
# https://school.programmers.co.kr/learn/courses/30/lessons/181886

def solution(names):
    answer = []

    temp = []
    for name in names:
        temp.append(name)

        if len(temp) == 5:
            answer.append(temp[0])
            temp.clear()

    if len(temp) != 0:
        answer.append(temp[0])

    return answer
